staging/operator-lifecycle-manager/test/e2e/ctx/ctx.go

```diff
@@ -111,8 +111,11 @@ func (ctx TestContext) NewE2EClientSession() {
 
 func (ctx TestContext) DumpNamespaceArtifacts(namespace string) error {
 	if ctx.artifactsDir == "" {
-		ctx.Logf("$ARTIFACTS_DIR is unset -- not collecting failed test case logs")
-		return nil
+		if os.Getenv("ARTIFACTS") == "" {
+			ctx.Logf("$ARTIFACTS_DIR is unset -- not collecting failed test case logs")
+			return nil
+		}
+		ctx.artifactsDir = os.Getenv("ARTIFACTS")
```

Comment:
It doesn't looks like this is a valid variable that the ci-operator injects: https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables.